### DISCUSSION TITLE
Unterminated string starting at: line 1 column 34 (char 33) : qute-keepassxc

### DISCUSSION THREAD

**Discussion by cronyakatsuki** [qutebrowser/qutebrowser#8059] [Q&A]
**Version info**:
https://bpa.st/KZKA

Wasn't able to test because of gpg error's.

**Description**

After month's of using keepassxc with the userscript to enter password's to my qutebrowser it just stopped working and started to show the error `Unterminated string starting at: line 1 column 34 (char 33)`. I'm not even sure how or why it happened, it might have been after adding one more password in, or removed it or anything like that, really not sure how or why I just know it stopped working all together and show's this error.

**How to reproduce**
Really don't know how to reproduce this error, I just know that after a long time of using this script it just stopped working out of the sudden and throws that error on any site I wan't to autocomplete the password.

**Comment by The-Compiler** [comment]
This seems like an issue with your setup somewhere and probably not a qutebrowser bug, so converting this to a discussion for now.

Where/how exactly is the error being shown? Would be good to have some more context, though personally I've never used that script.

**Reply by cronyakatsuki** [reply]
It shows as a message in the for notification's when I run it like a binding.

> related config line
```python
config.bind('pw', 'spawn --userscript qute-keepassxc --key KEY_MAIL')
```

Here I have a screenshot that show's me running the script and the message being shown.

![pic-full-240110-2128-07](https://github.com/qutebrowser/qutebrowser/assets/64900606/bc9e02db-b509-4813-b3a4-1fcb4b9cf4f1)

After entering the keybinding this error show's.

I used the script normally probably for a couple month's, when out of the sudden couple day's before it just stopped working and started showing the error.

**Reply by cronyakatsuki** [reply]
The error line is also visible when running qutebrowser from a terminal, it will get printed in STDOUT.

**Reply by cronyakatsuki** [reply]
Atleast I got one thing debuggeed thoroughly, it seem's that it doesn't work for specific domains and not all.

Will need to go through every website and try and find culprit.

If I do find it, will return what made the script error out so that you can make it a note in the script or something like that so other's know what to be carefull off when using the script.

**Reply by The-Compiler** [reply]
Could you try to reproduce this with `--temp-basedir` (might need to use the full path to the script with `:spawn -u`), and then do `:report` after this reproduces? This will let me take a look at the debug logs, which hopefully will say more.

**Reply by cronyakatsuki** [reply]
@The-Compiler I have managed to fix the issue in the end, it seems that if you have a few too many password's for the same domain like I do, but with differen't subfolder's or subdomain's the script can have a hard time.

The fix is to go to keepassxc settings > browser integration > and enable the option **return only best-matching result.**
After that the issue seem's like it has been fixed, you could add a note about that to the script, or I could make a pull request on that?
